packs/feats/ambush-bladderwort.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You plant the seed of a giant bladderwort in an unoccupied square of ground or water within 30 feet. It lasts for 10 minutes. Using this impulse again ends any previous one. If a creature enters its square, the seed erupts into a voluminous, translucent plant that seals tight around the triggering creature and fills with water. Unless the creature succeeds at a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC, it's @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] within the bladderwort and must hold its breath or begin drowning. The @UUID[Compendium.pf2e.actionspf2e.Item.Escape] DC is also your class DC. The plant has AC 10 and 50 Hit Points.</p>\n<p>If a creature dies inside it, the plant shrinks down, converting itself and the remains into a watery fruit. A creature can eat this consumable to regain [[/r (1d8+4)[healing]]] HP, after which that creature is temporarily immune for 10 minutes. This fruit rots after 1 hour.</p>\n<hr />\n<p><strong>Level (+4)</strong> The bladderwort's HP increase by 25 and the fruit's healing increases by 1d8+4.</p>"
+            "value": "<p>You plant the seed of a giant bladderwort in an unoccupied square of ground or water within 30 feet. It lasts for 10 minutes. Using this impulse again ends any previous one. If a creature enters its square, the seed erupts into a voluminous, translucent plant that seals tight around the triggering creature and fills with water. Unless the creature succeeds at a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC, it's @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] within the bladderwort and must hold its breath or begin drowning. The @UUID[Compendium.pf2e.actionspf2e.Item.Escape] DC is also your class DC. The plant has AC 10 and 50 Hit Points.</p>\n<p>If a creature dies inside it, the plant shrinks down, converting itself and the remains into a watery fruit. A creature can eat this consumable to regain @Damage[((floor((@actor.level - 4)/4)+1)d8+(floor((@actor.level - 4)/4)*4+4))[healing]] HP, after which that creature is temporarily immune for 10 minutes. This fruit rots after 1 hour.</p>\n<hr />\n<p><strong>Level (+4)</strong> The bladderwort's HP increase by 25 and the fruit's healing increases by 1d8+4.</p>"
```

Comment:
```suggestion
            "value": "<p>You plant the seed of a giant bladderwort in an unoccupied square of ground or water within 30 feet. It lasts for 10 minutes. Using this impulse again ends any previous one. If a creature enters its square, the seed erupts into a voluminous, translucent plant that seals tight around the triggering creature and fills with water. Unless the creature succeeds at a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC, it's @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] within the bladderwort and must hold its breath or begin drowning. The @UUID[Compendium.pf2e.actionspf2e.Item.Escape] DC is also your class DC. The plant has AC 10 and 50 Hit Points.</p>\n<p>If a creature dies inside it, the plant shrinks down, converting itself and the remains into a watery fruit. A creature can eat this consumable to regain @Damage[((floor((@actor.level - 4)/4)+1)d8+(floor((@actor.level - 4)/4)*4+4))[healing]] HP, after which that creature is temporarily immune for 10 minutes. This fruit rots after 1 hour.</p>\n<hr />\n<p><strong>Level (+4)</strong> The bladderwort's HP increase by 25 and the fruit's healing increases by 1d8+4.</p>"
```

Inline healing isn't ready to be used for `@Damage` yet. We inadvertently caught it in some changes we made before but have since undone those.

Reply:
With the formulas is not being parsed correctly, should I put “leveled HP” or leave it as is?
![image](https://github.com/foundryvtt/pf2e/assets/29728598/82e71ad7-b6ce-45e8-a31d-db0efe49ae77)
![image](https://github.com/foundryvtt/pf2e/assets/29728598/0fee5dd7-7400-4afa-9743-c27a4013bc2e)

Reply:
Put the baseline in the description and a leveled button with a label after that paragraph.

Reply:
Changed.